src/main/java/com/conveyal/r5/profile/FastRaptorWorker.java

```diff
@@ -141,26 +142,17 @@ public FastRaptorWorker (TransitLayer transitLayer, ProfileRequest request, TInt
         this.request = request;
         this.accessStops = accessStops;
         this.servicesActive  = transit.getActiveServicesForDate(request.date);
-        // we add one to request.maxRides, first state is result of initial walk
-        this.scheduleState = IntStream.range(0, request.maxRides + 1)
-                .mapToObj((i) -> new RaptorState(
-                            transit.getStopCount(),
-                            request.maxTripDurationMinutes * SECONDS_PER_MINUTE))
-                .toArray(RaptorState[]::new);
-
-        for (int i = 1; i < this.scheduleState.length; i++) {
-            this.scheduleState[i].previous = this.scheduleState[i - 1];
-        }
 
         offsets = new FrequencyRandomOffsets(transitLayer);
 
         // compute number of minutes for scheduled search
         nMinutes = request.getTimeWindowLengthMinutes();
 
-        // how many monte carlo draws per minute of scheduled search to get desired total iterations?
+        // How many random schedules per departure minute to meet or exceed the requested total iterations?
         monteCarloDrawsPerMinute = request.getMonteCarloDrawsPerMinute();
 
         // Zero Monte Carlo draws means use half-headway instead of Monte Carlo randomization.
+        // TODO review this logic and parameter flow
         boardingMode = (monteCarloDrawsPerMinute == 0) ? HALF_HEADWAY : MONTE_CARLO;
```

Comment:
Is this something that should be more explicit? 

This makes it appear that setting the monteCarloDraws to a number < 120 would turn on HALF_HEADWAY mode.

Reply:
The requested number of Monte Carlo draws is a lower limit. The actual number may be larger, corresponding to a positive integer number of draws per minute. A monteCarloDraws in the range [1...120] should give 1 per minute, or 120 draws. I will check that logic though before merging this.

Reply:
Yes, I meant should setting this mode be more explicit, both here and in the UI. If a user reduces the amount of draws and it triggers this new mode.

In fact, with the ability to set monte carlo draws in the JSON. Should we even expose simulated schedules text field as a UI element? Should we instead expose "modes" that correlate to these options that instead?

Reply:
This conversation should not hold up this PR, as this PR only adds a comment here.

Reply:
Ah, gotcha. I misinterpreted the code here and this shows my unfamiliarity with this part of the system. I had assumed that minutes = 120 and the simulated schedules set in the UI was a total across an analysis, not the total per minute.

Reply:
Simulated schedules set in the UI is the total for the whole analysis, across all minutes. The total per minute is the value set in the UI divided by the number of minutes and rounded up.